docs/manual/notebookashomepage.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: page
+title: "Notebook as Homepage"
+description: ""
+group: manual
+---
+{% include JB/setup %}
+
+## Customize your zeppelin homepage
+ Zeppelin allows you to use one of the notebooks you create as your zeppelin Homepage.
+ With that you can brand your zeppelin installation, 
+ adjust the instruction to your users needs and event translate to other languages.
```

Comment:
fixed the typo in the doc.
Thanks!

On Mon, Aug 17, 2015 at 11:15 AM Felix Cheung <notifications@github.com>
wrote:

> In docs/manual/notebookashomepage.md
> <https://github.com/apache/incubator-zeppelin/pull/214#discussion_r37166194>
> :
>
> > @@ -0,0 +1,96 @@
> > +---
> > +layout: page
> > +title: "Notebook as Homepage"
> > +description: ""
> > +group: manual
> > +---
> > +{% include JB/setup %}
> > +
> > +## Customize your zeppelin homepage
> > + Zeppelin allows you to use one of the notebooks you create as your zeppelin Homepage.
> > + With that you can brand your zeppelin installation,
> > + adjust the instruction to your users needs and event translate to other languages.
>
> "event" -> even?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-zeppelin/pull/214/files#r37166194>.
>